src/components/Button/Button.styled.ts

```diff
@@ -209,20 +209,20 @@ const textButtonStyles = {
 export const types = {
 	yellow_contained: styled(BlackOnYellow)`
     & > button {
-      background-color: ${theme.colors.simplyGold};
+      background-color: ${theme.newColors.simplyGold["100"]};
       border-radius: 0;
       font-size: 14px;
       text-transform: uppercase;
     }
 
     .MuiButton-contained.Mui-disabled {
       color: ${theme.colors.almostBlack};
-      background-color: ${theme.colors.simplyGold};
+      background-color: ${theme.newColors.simplyGold["100"]};
       opacity: 0.5;
     }
 
     & > button:hover {
-      background-color: ${theme.colors.simplyGoldHover};
+      background-color: ${theme.newColors.simplyGold["100"]};

```

Comment:
I think this should be another color. Comparing the theme file and the figma, the color for simplyGoldHover is the same as the darkerSimplyGold[100], let's keep it as simplyGoldHover or change it to newColors.darkerSimplyGold['100'] please.